### PR TITLE
Fix wrong variable in EmbedBuilder#addField() docs

### PR DESCRIPTION
**Pull request by napstr**

### CODE REVIEW

**Comment by MinnDevelopment** [comment]
Can we change these to a `{@value ...}` tags?

**Comment by napstr** [comment]
Done.

**Comment by MinnDevelopment** [comment]
Could you do it for all the limit references in the other respective methods as well?

**Comment by napstr** [comment]
Done.

File: src/main/java/net/dv8tion/jda/core/entities/MessageEmbed.java [hunk 1]
```diff
@@ -156,8 +156,8 @@
      * <p>An Embed can only have, at max, {@value #EMBED_MAX_LENGTH_BOT} displayable text characters for {@link net.dv8tion.jda.core.AccountType#BOT AccountType.BOT}
      * accounts or {@value #EMBED_MAX_LENGTH_CLIENT} displayable text characters for {@link net.dv8tion.jda.core.AccountType#CLIENT AccountType.CLIENT} accounts.
      *
-     * <p>Both of these values are defined by {@link #EMBED_MAX_LENGTH_BOT EMBED_MAX_LENGTH_BOT} and
-     * {@link #EMBED_MAX_LENGTH_CLIENT EMBED_MAX_LENGTH_CLIENT} respectively.
+     * <p>Both of these values are defined by {@value #EMBED_MAX_LENGTH_BOT EMBED_MAX_LENGTH_BOT} and
+     * {@value #EMBED_MAX_LENGTH_CLIENT EMBED_MAX_LENGTH_CLIENT} respectively.
```
Comment (MinnDevelopment) [inline src/main/java/net/dv8tion/jda/core/entities/MessageEmbed.java]: Value tags don't support alias

Comment (napstr) [inline src/main/java/net/dv8tion/jda/core/entities/MessageEmbed.java]: Rip. Force push inc.

**Comment by Almighty-Alpaca** [comment]
nice mentions :eyes:

**Comment by napstr** [comment]
Reverted to the original commit which just fixes the error.

**Pull request merged**